### PR TITLE
Zib-ContactInformation

### PR DESCRIPTION
**Pull request by AmyKoo1**

### CODE REVIEW

**Comment by LuudSlagter** [comment]
Note: I have changed the `.version` of the manual ValueSets from '4.0.1' (in 2020) to '2024-04-15T00:00:00' (equal to the generated ValueSets), since the former represents the FHIR version.

**Comment by LuudSlagter** [comment]
Only thing that needs to be done before merging is manually correcting the ValueSets with extensible binding (i.e. removing OTH) and refreshing the terminology afterwards.

**Comment by LuudSlagter** [comment]
Known issues related to effective extension cardinalities will be fixed later on via https://nictiz.atlassian.net/browse/ZIBFHIR2024-363.

File: zib/vocabulary/manual/ValueSet-ContactInformation-TelephoneNumbersUse-EmergencyContact.xml [hunk 1]
```diff
@@ -19,7 +18,7 @@
             <system value="http://terminology.hl7.org/CodeSystem/v3-AddressUse"/>
             <concept>
                 <code value="EC"/>
-                <display value="Emergencycontact"/>
+                <display value="Emergency contact"/>
```
Comment (ArdonToonstra) [inline zib/vocabulary/manual/ValueSet-ContactInformation-TelephoneNumbersUse-EmergencyContact.xml:21]: The actual display value is without a: "emergency contact" 
https://terminology.hl7.org/6.1.0/CodeSystem-v3-AddressUse.html

**Review by ArdonToonstra — DISMISSED** [review]

File: zib/vocabulary/manual/ValueSet-ContactInformation-TelephoneNumbersUse-EmergencyContact.xml (hunk 1 above)
Comment (LuudSlagter) [reply zib/vocabulary/manual/ValueSet-ContactInformation-TelephoneNumbersUse-EmergencyContact.xml:21]: Based on ZIB-1436 and the fact that we have manually created subsets of the zib ValueSet NummerSoortCodelijst (in which capitals are used), Ardon and I concluded that it's better to keep the `display` as-is.

**Review by pieter-edelman-nictiz — APPROVED** [review]

**Pull request merged**